internal/format/format_test.go

```diff
@@ -0,0 +1,89 @@
+package format
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var writer = bytes.NewBuffer(nil)
+
+var spec []*formatSpec = []*formatSpec{
+	{
+		Formats: []*formatWithSpected{
+			{
+				Format: NewCsvFormat(writer),
+				Result: "a,b,c\na1,b1,c1\n",
+			}, {
+				Format: NewJsonFormat(writer),
+				Result: "{\"a\":\"a1\",\"b\":\"b1\",\"c\":\"c1\"}\n",
+			}, {
+				Format: NewPrettyFormat(writer),
+				Result: "+----+----+----+\n| A  | B  | C  |" +
+					"\n+----+----+----+\n| a1 | b1 | c1 |" +
+					"\n+----+----+----+\n",
+			},
+		},
+		Headers: []string{"a", "b", "c"},
+		Lines: [][]interface{}{
+			[]interface{}{
+				"a1", "b1", "c1",
+			},
+		},
+	},
+}
+
+func TestFormats(t *testing.T) {
```

Comment:
use separate functions for each format so that we have separate test output for them